compiler/src/dotty/tools/backend/jvm/BCodeBodyBuilder.scala

```diff
@@ -440,10 +464,10 @@ trait BCodeBodyBuilder extends BCodeSkelBuilder {
      * must-single-thread
      */
     private def fieldOp(field: Symbol, isLoad: Boolean, specificReceiver: Symbol): Unit = {
-      val useSpecificReceiver = specificReceiver != null && !field.isScalaStatic
+      val useSpecificReceiver = specificReceiver != null && !field.hasAnnotation(ctx.definitions.ScalaStaticAnnot)
 
```

Comment:
Since this comes up several times, I think it would be good to add `isScalaStatic` to SymUtils.

Reply:
Done